packages/react-resizable-panels-website/playwright.config.ts

```diff
@@ -14,11 +14,13 @@ const config: PlaywrightTestConfig = {
     },
   ],
   reporter: [
-    // @ts-ignore
-    createReplayReporterConfig({
-      apiKey: process.env.REPLAY_API_KEY,
-      upload: true,
-    }),
+    process.env.REPLAY_API_KEY
```

Comment:
alternatively, you could fork the behavior based on `if: github.repository == 'bvaughn/react-resizable-panels'` in the workflow file